html/index.html

```diff
@@ -1205,4 +1205,7 @@
       gtag('config', 'G-WMZZXH53QH');
     </script>
   </body>
+  <footer>
+    <button id="pmLink">Privacy Manager</button>
```

Comment:
@iolayemi , just out of curiosity, are there any handlers assigned to this button? how does it work when the user click on it? is it managed by some snippet embedded?

Reply:
That is the question I asked Dan during standup. 
My understanding is that there is a library embedded which is responsible for targeting this button and styling

Reply:
I see. ok, sounds good, thank you Olayemi